build/index.html

```diff
@@ -0,0 +1,6 @@
74,2251,1406,3337,1146,5,2,2,1968,2590,182,283,556,2580,669,1050,1808,1129,268,81,7,1,2,25,463,620,29,983,6,406,458,1847,769,536,196,499,3395,1209,876,412,2,3188,381,438,796,1220,38,363,557,270,448,155,1217,956,408,484,47,1080,542,1166,984,44,1558,1245,258,2,631,217,920,2,1128,295,2,4,2,140,530,44,334,682,372,381,418,349,167,82,247,1116,635,1592,1015,12,620,464,389,283,278,44,227,78,26,330,639,126,16,45,276,174,61,20,2,130,293,2493,530,206,22,587,366,457,366,70,3,308,306,145,208,216,197,135,65,20,1,12,392,19,11,390,49,30,249,154,320,24,175,14,167,271,4,256,289,71,103,110,85,130,131,70,5962426,1874,680,95,140,19,3,8797529,4,249,1323,549,333,444,1,2,80,1,900,583,7,306,1,8,1,2,2132,1,1,1,1,1,414,1,748,141,59,726,3,7,563,1,616,22,23,39,7,40,22306391',authuser:0,kscs:'c9c918f0_BbpcXLnQKqWJ0gKt647wDw',kGL:'US'};google.kHL='en';})();google.time=function(){return(new Date).getTime()};(function(){google.lc=[];google.li=0;google.getEI=function(a){for(var b;a&&(!a.getAttribute||!(b=a.getAttribute("eid")));)a=a.parentNode;return b||google.kEI};google.getLEI=function(a){for(var b=null;a&&(!a.getAttribute||!(b=a.getAttribute("leid")));)a=a.parentNode;return b};google.https=function(){return"https:"==window.location.protocol};google.ml=function(){return null};google.log=function(a,b,e,c,g){if(a=google.logUrl(a,b,e,c,g)){b=new Image;var d=google.lc,f=google.li;d[f]=b;b.onerror=b.onload=b.onabort=function(){delete d[f]};google.vel&&google.vel.lu&&google.vel.lu(a);b.src=a;google.li=f+1}};google.logUrl=function(a,b,e,c,g){var d="",f=google.ls||"";e||-1!=b.search("&ei=")||(d="&ei="+google.getEI(c),-1==b.search("&lei=")&&(c=google.getLEI(c))&&(d+="&lei="+c));c="";!e&&google.cshid&&-1==b.search("&cshid=")&&"slh"!=a&&(c="&cshid="+google.cshid);a=e||"/"+(g||"gen_204")+"?atyp=i&ct="+a+"&cad="+b+d+f+"&zx="+google.time()+c;/^http:/i.test(a)&&google.https()&&(google.ml(Error("a"),!1,{src:a,glmm:1}),a="");return a};}).call(this);(function(){google.y={};google.x=function(a,b){if(a)var c=a.id;else{do c=Math.random();while(google.y[c])}google.y[c]=[a,b];return!1};google.lm=[];google.plm=function(a){google.lm.push.apply(google.lm,a)};google.lq=[];google.load=function(a,b,c){google.lq.push([[a],b,c])};google.loadAll=function(a,b){google.lq.push([a,b])};}).call(this);google.f={};</script><script nonce="cV6G+uNGqI2mF3HWdzPVOg==">var a=window.location,b=a.href.indexOf("#");if(0<=b){var c=a.href.substring(b+1);/(^|&)q=/.test(c)&&-1==c.indexOf("#")&&a.replace("/search?"+c.replace(/(^|&)fp=[^&]*/g,"")+"&cad=h")};</script><style>#gbar,#guser{font-size:13px;padding-top:1px !important;}#gbar{height:22px}#guser{padding-bottom:7px !important;text-align:right}.gbh,.gbd{border-top:1px solid #c9d7f1;font-size:1px}.gbh{height:0;position:absolute;top:24px;width:100%}@media all{.gb1{height:22px;margin-right:.5em;vertical-align:top}#gbar{float:left}}a.gb1,a.gb4{text-decoration:underline !important}a.gb1,a.gb4{color:#00c !important}.gbi .gb4{color:#dd8e27 !important}.gbf .gb4{color:#900 !important}
```

Comment:
I've got to ask - why do we have... what looks like a copy of the Google homepage in here? :confused: :smile:

Reply:
Honestly I have no memory of explicitly adding this! It must have been some git mistake on my side. Removed.